### ISSUE TITLE
Path to cuda

### ISSUE THREAD

**Issue by MariaMel98** [viscom-ulm/MCCNN#10]
Hello, I'm a student and I'm interested in your code.  I tried to compile tensorflow operations by following your instructions, but I get an error. The server that I'm working on has different versions of cuda installed but when I  enter **whereiscuda** on the terminal it shows "usr/local/cuda" so I'm using this path. Is this correct? Or should the path be "usr/local/cuda-11.2" since tensorflow works with that specific version?

**Comment by phermosilla** [comment]
Hello, thanks for your interest in our code.  This code is quite old and I did not try it with this version of Cuda. But it seems that you are doing everything correctly. However, I would recommend you to use a newer implementation of this work in Pytorch that you can find in the following repository: 

https://github.com/phermosilla/pccnn_weightinit_experiments

I am still working on the tutorials, etc. But it should be updated in the following weeks before ECCV at the end of October.

**Comment by MariaMel98** [comment]
Does this new code works the same as MCCNN Library? Because I'm trying to use this code [https://github.com/phermosilla/TotalDenoising ](url) and it needs MCCNN.

**Comment by phermosilla** [comment]
No, sorry. You will need to use this library for the code in total denoising. I recommend then using a docker container where everything is setup properly.
